consensus/parlia/parlia.go

```diff
@@ -709,6 +1063,12 @@ func (p *Parlia) Finalize(chain consensus.ChainHeaderReader, header *types.Heade
 	if err != nil {
 		return err
 	}
+
+	if p.chainConfig.IsLynn(header.Number) {
```

Comment:
==> `p.chainConfig.IsLynn(header.Number-1)`

Reply:
We discussed about this. It's ok to use number rather than number-1 as the related contracts have already updated after boneh